static/app/views/performance/traceDetails/traceViewDetailPanel.tsx

```diff
@@ -0,0 +1,504 @@
+import {DataSection} from 'sentry/components/events/styles';
+import FileSize from 'sentry/components/fileSize';
+import ProjectBadge from 'sentry/components/idBadge/projectBadge';
+import Link from 'sentry/components/links/link';
+import LoadingIndicator from 'sentry/components/loadingIndicator';
+import {
+  ErrorDot,
+  ErrorLevel,
+  ErrorMessageContent,
+  ErrorMessageTitle,
+  ErrorTitle,
+} from 'sentry/components/performance/waterfall/rowDetails';
+import QuestionTooltip from 'sentry/components/questionTooltip';
+import {generateIssueEventTarget} from 'sentry/components/quickTrace/utils';
+import {Tooltip} from 'sentry/components/tooltip';
+import {PAGE_URL_PARAM} from 'sentry/constants/pageFilters';
+import {IconChevron, IconOpen} from 'sentry/icons';
+import {t, tn} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import {EntryBreadcrumbs, EntryType, EventTransaction, Organization} from 'sentry/types';
+import {objectIsEmpty} from 'sentry/utils';
+import {trackAnalytics} from 'sentry/utils/analytics';
+import getDynamicText from 'sentry/utils/getDynamicText';
+import {PageErrorProvider} from 'sentry/utils/performance/contexts/pageError';
+import {generateProfileFlamechartRoute} from 'sentry/utils/profiling/routes';
+import {useLocation} from 'sentry/utils/useLocation';
+import useOrganization from 'sentry/utils/useOrganization';
+import useProjects from 'sentry/utils/useProjects';
+import {isCustomMeasurement} from 'sentry/views/dashboards/utils';
+import DetailPanel from 'sentry/views/starfish/components/detailPanel';
+
+import {transactionSummaryRouteWithQuery} from '../transactionSummary/utils';
+
+import {EventDetail} from './newTraceDetailsContent';
+import {Row, Tags} from './styles';
+
+type DetailPanelProps = {
+  detail: EventDetail | undefined;
+  onClose: () => void;
+};
+
+type EventDetailProps = {
+  detail: EventDetail;
+  location: Location;
+  organization: Organization;
+};
+
+function OpsBreakdown({event}: {event: EventTransaction}) {
+  const [showingAll, setShowingAll] = useState(false);
+  const breakdown = event && generateStats(event, {type: 'no_filter'});
+
+  if (!breakdown) {
+    return null;
+  }
+
+  const renderText = showingAll ? t('Show less') : t('Show more') + '...';
+  return (
+    breakdown && (
+      <Row
+        title={
+          <FlexBox style={{gap: '5px'}}>
+            Ops Breakdown
+            <QuestionTooltip
+              title={t('Applicable to the children of this event only')}
+              size="xs"
+            />
+          </FlexBox>
+        }
+      >
+        <div style={{display: 'flex', flexDirection: 'column', gap: space(0.25)}}>
+          {breakdown.slice(0, showingAll ? breakdown.length : 5).map((currOp, index) => {
+            const {name, percentage, totalInterval} = currOp;
+
+            const operationName = typeof name === 'string' ? name : t('Other');
+            const durLabel = Math.round(totalInterval * 1000 * 100) / 100;
```

Comment:
I'm a little unclear here why we also multiply by 100, can we leave a comment r.e. what we're converting? I'm familiar with multiplying by 1000 to get ms but I could use a bit more clarity here

Reply:
Maybe we could use the `PerformanceDuration` component here? It takes in some consistent time prop (e.g. milliseconds) and spits out a nice representation with the label you're adding in line 105

Reply:
Yup, the PerformanceDuration component works here.